package.json

```diff
@@ -21,8 +21,9 @@
     "pack:zip": "rimraf mullvad-browser-extension-$npm_package_version.zip && jszip-cli add extension -o ./mullvad-browser-extension-$npm_package_version.zip",
     "pack:xpi": "cross-env WEB_EXT_ARTIFACTS_DIR=./ web-ext build --source-dir ./extension --filename mullvad-browser-extension-$npm_package_version.xpi --overwrite-dest",
     "start:chromium": "web-ext run --source-dir ./extension --target=chromium",
-    "start": "web-ext run --source-dir ./extension --target=firefox-desktop --start-url=about:debugging#/runtime/this-firefox",
-    "restart": "web-ext run --firefox-profile ~/.mozilla/firefox/md0hl4g1.MBE-Testing-Profile --keep-profile-changes --source-dir ./extension --target=firefox-desktop --start-url=about:debugging#/runtime/this-firefox",
+    "start": "web-ext run --source-dir ./extension --target=firefox-desktop --start-url=about:debugging#/runtime/this-firefox --start-url=mullvad.net/check",
```

Comment:
Is it possible to use multiple `--start-url`?

Reply:
Yes, multiple tabs will open.